doc/getting-started/getting-started.rst

```diff
@@ -637,9 +637,9 @@ In the following steps, you will configure GRUB on the target system.
            insmod gzio
            insmod part_gpt
            insmod ext2
-           search --no-floppy --fs-uuid --set <UUID>
+           search --no-floppy --fs-uuid --set "3cac5675-e329-4cal-b346-0a3e65f99016"
```

Comment:
This isn't going to work.  The instructions above this code block say to

```
Replace ``<UUID>`` and ``<PARTUUID>`` with the output from the previous step.
```
but now the code block doesn't have those strings in it any more.  What we need is to put those UUID and PARUUID strings back in this code block, and then create another code block showing what a properly edited GRUB file example would look like.

Reply:
already edited